pkgs/top-level/all-packages.nix

```diff
@@ -2235,6 +2235,8 @@ in
 
   bash-supergenpass = callPackage ../tools/security/bash-supergenpass { };
 
+  swappy = callPackage ../applications/misc/swappy { gtk = gtk3; };
```

Comment:
flameshot, a similar tool is in pkgs/tools/misc/flameshot. Not sure if it's important.